intel_extension_for_transformers/neural_chat/examples/instruction_tuning/README.md

```diff
@@ -14,12 +14,35 @@ This example demonstrates how to finetune the pretrained large language model (L
 # Prerequisite​
 
 ## 1. Environment​
+### Bare Metal
 Recommend python 3.9 or higher version.
 ```shell
 pip install -r requirements.txt
 # To use ccl as the distributed backend in distributed training on CPU requires to install below requirement.
 python -m pip install oneccl_bind_pt==1.13 -f https://developer.intel.com/ipex-whl-stable-cpu
```

Comment:
This command isn't working for me. Tried 1.13.0 and 1.13.0+cpu but also doesn't work.

Reply:
I don't change oneccl command. it also failed on me. I need to download it and install the wheel instead. need to create a JIRA accordingly.

Reply:
is it a compatibility issue between torch and ccl?